src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/entities/EntityRecognizer.scala

```diff
@@ -39,8 +39,6 @@ class EntityRecognizer(
       OpeNER.posAnnotate(resourcesDirectory, language.get, kaf)
       OpeNER.nerAnnotate(resourcesDirectory, language.get, kaf)
 
-      logDebug(s"Analyzed text $text in language $language: $kaf")
-
```

Comment:
The implicit call to `kaf.toString` was occasionally causing a NullPointerException, so I removed this debug call for now.